website/content/v1.9/talos-guides/configuration/image-cache.md

```diff
@@ -36,14 +36,17 @@ There are two supported boot asset types for the Image Cache: ISO and disk image
 
 ### ISO
 
+> Note: ISO image cache only works with iso9660 filesystems.
```

Comment:
what does it mean by an `iso9660`, I guess virtual cdroms are not so much used as an option, also newer ISO support something called File System Transposition and are not really supposed to be `dd`'ed to a disk

Reply:
some readup:

* https://github.com/pbatard/rufus/wiki/FAQ#user-content-Why_doesnt_Rufus_recommend_DD_mode_over_ISO_mode_for_ISOHybrid_images_Surely_DD_is_better
* https://github.com/pbatard/rufus/issues/2534#issuecomment-2263949687

Reply:
I think we need to fix this one way or another, but I'm confused a bit about the method. We should document what is the supported method of moving an ISO to a USB stick, and the one we support here

Reply:
ok, we have a plan going to support two things:

* `dd` the ISO to the USB stick (already works without issues)
* copy the ISO contents to the VFAT usb stick (need some changes to support that)

Reply:
I was basing this off what I was seeing on my physical hardware in the various tests. It only seemed to work when the detected filesystem was iso9660 but I didn't exhaustively test what's available.

Reply:
> * `dd` the ISO to the USB stick (already works without issues)

When I `dd` the iso to usb the usb drive showed up as sda1 and I don't think the cache was used during provisioning. I can test again if you think this should work.